themes/barbuk/README.rst

```diff
@@ -0,0 +1 @@
+../../docs/themes-list/barbuk.rst
```

Comment:
no need for this file anymore

Reply:
It's a symlink to keep displaying the documentation in github source.
I can remove it, but I think it can be useful for people browsing the source code.

Reply:
The references are broken though.. github do not render them properly, so I am not sure it is really optimal. If you say it helps you, we can keep it

Reply:
> The references are broken though.. 

Indeed, but I don't think it's an issue.

Having the documentation directly visible in the source directory (as a readme in the IDE and in github) makes it more visible.

>  If you say it helps you, we can keep it

Perfect. I'll mark the conversation as resolved.

PS: Merry Christmas !